packages/integration-karma/test/component/LightningElement.render/index.spec.js

```diff
@@ -70,29 +70,3 @@ it('supports returning different templates', () => {
         expect(elm.shadowRoot.textContent).toBe('Template 2');
     });
 });
-
-// The error in this test case happens in a Promise chain that is not within the control of the test.
-// Hence the error has to be caught at the top level
-if ('onunhandledrejection' in window) {
-    describe('should throw error during async rehydration', () => {
```

Comment:
just curious, why we can remove this now?

Reply:
Its that "we can" remove it now, its a matter of "we have to remove it now" because of PM's comment in the PR description `Since jasmine 3.6, all the unhandledrejections events force the test to fail.`
The test relies on being able to catch the promise rejection at the window and do the assertion.